ui/app/pages/swaps/actionable-message/index.scss

```diff
@@ -57,9 +64,40 @@
   &--left-aligned {
     .actionable-message__message,
     .actionable-message__actions {
+    }
+  }
+
+  &--with-right-button {
+    padding: 12px;
+
+    .actionable-message__message {
       justify-content: flex-start;
       text-align: left;
       width: 100%;
     }
+
+    .actionable-message__actions {
+      justify-content: flex-end;
+      width: 100%;
+    }
+
+    .actionable-message__action {
+      font-weight: normal;
+      cursor: pointer;
```

Comment:
Nit: This rule is redundant. It already has `cursor: pointer` from the base `actionable-message__action` class

Reply:
This can get cleaned up in the PR that makes this more accessible.